hack/kind-cluster-build.sh

```diff
@@ -258,14 +256,29 @@ EOF
 kubectl apply -f ${registryFile}
 
 echo "init tidb-operator env"
-kubectl apply -f manifests/local-dind/kube-flannel.yaml
 kubectl apply -f manifests/local-dind/local-volume-provisioner.yaml
```

Comment:
```
error: the path "manifests/local-dind/local-volume-provisioner.yaml" does not exist
```
The execution result of this script is related to the execution location. We should define a `WORKSPACE` variable that is location-agnostic. Then use this variable and relative path to assemble into an absolute path.  For example:
```bash
WORKSPACE=$(cd $(dirname $0)/..; pwd)
kubectl apply -f ${WORKSPACE}/manifests/local-dind/local-volume-provisioner.yaml
...
```

Reply:
good suggestion!

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/890/commits/50e369bbf48de084a6c0a07b3103c762155effab
cd to the root of the repo, then the script can be executed in any working directroy